Documentation/Storage-Configuration/Advanced/configuration.md

```diff
@@ -20,11 +20,31 @@ configuration options which users are well advised to consider for any productio
 
 ### Default PG and PGP counts
 
-The number of PGs and PGPs can be configured on a per-pool basis, but it is highly advised to set
-default values that are appropriate for your Ceph cluster. Appropriate values depend on the number
-of OSDs the user expects to have backing each pool. The Ceph [OSD and Pool config
+The number of PGs and PGPs can be configured on a per-pool basis, but it is
+highly advised to set default values that are appropriate for your Ceph
+cluster. Appropriate values depend on the number of OSDs the user expects to
+have backing each pool. These can be configured by declaring pg_num and pgp_num
+parameters under CephBlockPool resource.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: ceph-block-pool-test
+  namespace: rook-ceph
+spec:
+  deviceClass: hdd
+  replicated:
+    size: 3
+spec:
+  parameters:
+    pg_num: '4' # create the pool with a pre-configured placement group number
```

Comment:
Why is `4` chosen in this example? perhaps a brief comment would help to explain why this number is used in the example.

Reply:
good point! added relevant reference for pgcaluation and changed this to a small example.